BE/src/main/java/todo3/codesquad/service/TodoService.java

```diff
@@ -0,0 +1,206 @@
+            }
+            return card;
+        }
+
+        List<Card> oriCards = colRepository.findCardsByColName(originColName).orElse(null);
+        List<Card> desCards = colRepository.findCardsByColName(destinationColName).orElse(null);
+        Card movedCard = colRepository.findCardByCardId(moveCardId).orElse(null);
+        for (int i = 0; i < oriCards.size(); i++) {
+            Long cardId = oriCards.get(i).getId();
+            if (cardId.equals(moveCardId)) {
+                oriCards.remove(i);
+            }
+        }
+        if (destinationRow == 99) {
+            desCards.add(movedCard);
+        } else {
+            desCards.add(destinationRow - 1, movedCard);
+        }
+        if (!newCardsSetColumn(oriCards, originCol.getColName()) || !newCardsSetColumn(desCards, destinationColName)) {
+            return null;
+        }
+        return movedCard;
+    }
+
+    private boolean newCardsSetColumn(List<Card> cards, String columnName) {
+        List<Card> newCards = cardsRowReset(cards);
+        Col column = colRepository.findColByColName(columnName).orElse(null);
+        if (column == null) {
+            return false;
+        }
+        column.setCards(newCards);
+        colRepository.save(column);
+        return true;
+    }
+
+    public List<Card> cardsRowReset(List<Card> cards) {
+        int rowIdx = 1;
+        for (int i = 0; i < cards.size(); i++) {
+            Card tempCard = cards.get(i);
+            if (tempCard.getDeleted() || tempCard.getRow() == 0) {
+                continue;
+            }
+            tempCard.setRow(rowIdx);
+            rowIdx++;
+        }
+        return cards;
+    }
+
+    public Card deleteCard(Long deleteCardId) {
+        log.info("Deleted Card ID----------------{}",deleteCardId );
+        if (colRepository.deleteCardByCardId(deleteCardId)) {
+            Col col = colRepository.findColByCardId(deleteCardId).orElse(null);
+            List<Card> cards = col.getCards();
+            int rowIdx = 1;
+            for (int i = 0; i < cards.size(); i++) {
+                if (cards.get(i).getDeleted()) {
+                    continue;
+                }
+                cards.get(i).setRow(rowIdx);
+                rowIdx++;
+            }
+            colRepository.save(col);
+            return colRepository.findCardByCardId(deleteCardId).orElse(null);
+        }
+        return null;
+    }
+
+    public List<Col> showColumns() {
+        List<Col> columns = colRepository.findColByNotDeleted().orElse(null);
+        List<Col> newColumns = new ArrayList<>();
+        for (int i = 0; i < columns.size(); i++) {
+            Col tempCol = columns.get(i);
+            Long colId = tempCol.getId();
+            List<Card> tempCards = colRepository.findNotDeletedCardsByColId(colId).orElse(null);
+            if (tempCol.isDeleted()) {
+                continue;
+            }
+            tempCol.setCards(tempCards);
+            newColumns.add(tempCol);
```

Comment:
continue를 사용하지 않은지 너무 오래되어 헷갈리는데, 꼭 continue를 사용하지 않아도 괜찮은 상황으로 보이네요.
개인적인 취향이지만 최대한 키워드를 적게 사용하고 indent를 줄이는 것이 가독성에 조금 더 도움을 주는 것 같습니다.

```suggestion
            if (!tempCol.isDeleted()) {
                tempCol.setCards(tempCards);
                newColumns.add(tempCol);
            }
```